src/components/mdx/md-contents.tsx

```diff
@@ -38,7 +38,7 @@ export const styleOverwrites = {
     pre: {
       // px: space(['none', 'none', 'extra-loose', 'extra-loose']),
     },
-    img: {
+    '.img': {
```

Comment:
Why does it need the `.`?

Reply:
previously the spacing on images was by tag, but we want to treat the top level element as the spacing control, and with the aspect ratio changes, the top level element can sometimes be a `div`, so both have the className `img`.